src/main/scala/cromwell/engine/db/slick/RunMysql.scala

```diff
@@ -141,8 +141,8 @@ object RunMysql {
     getConfigOpt("properties." + key) orElse getConfigOpt(key) getOrElse ""
 
   private def getConfigOpt(key: String): Option[String] = {
```

Comment:
I'm inclined to delete this entire file. It was used for debugging a while ago. I only updated the minimum to keep it compiling. I doubt it even runs. Unless I hear any objections, it shall retire to the git archives.